src/gbt_model_builder.pyx

```diff
@@ -179,3 +179,33 @@ def gbt_reg_model_builder(n_features, n_iterations):
     :param size_t n_iterations: Number of trees in the model
     '''
     return gbt_regression_model_builder(n_features, n_iterations)
+
+class GBTModel():
+    '''
+    Daal4py GBT Model
+    '''
+
+    def __init__(self, model):
+        framework_name = (model.__class__.__module__, model.__class__.__name__)
+        if framework_name == ("lightgbm.basic", "Booster"):
+            self._model, self._n_classes = get_gbt_model_from_lightgbm(model)
+        elif framework_name == ("xgboost.core", "Booster"):
```

Comment:
Support of xgb.XGBClassifier/Regressor is required. Same for other frameworks if they contain sklearn-like estimators